layersvt/screenshot.cpp

```diff
@@ -254,11 +267,32 @@ void readScreenShotFormatENV(void) {
                                 "SRGB\nSwapchain Colorspace will be used instead\n",
                                 vk_screenshot_format);
 #else
-            fprintf(stderr, "Selected format:%s\nIs NOT in the list:\nUNORM, SNORM, USCALED, SSCALED, UINT, SINT, SRGB\n"
-                            "Swapchain Colorspace will be used instead\n", vk_screenshot_format);
+            fprintf(stderr,
+                    "Selected format:%s\nIs NOT in the list:\nUNORM, SNORM, USCALED, SSCALED, UINT, SINT, SRGB\n"
+                    "Swapchain Colorspace will be used instead\n",
+                    vk_screenshot_format);
 #endif
         }
     }
+
+    if (env_var != NULL && strlen(env_var) > 0) {
```

Comment:
In the case of an empty string as an environment variable, wouldn't it be best to use the vk_layers_settings.txt file, just in case the user accidentally set the environment variable to an empty string instead of unsetting it?